src/App.js

```diff
@@ -3,49 +3,56 @@ import Search from "./components/Search";
 import Basket from "./components/Basket/Basket";
 import { useState } from "react";
 import Footer from "./components/Footer";
-
+import CatsIcon from "./cat.png";
+import CartIcon from "./cart.png";
+import SearchIcon from "./searchlogo.png";
 
 function App() {
-
   // State to store the cats in the basket
 
   const [basketItems, setBasketItems] = useState([]);
   const [basketVisible, setBasketVisible] = useState(false);
 
-  const toggleBasket=()=> {
+  const toggleBasket = () => {
     //Toggle the basket
-    setBasketVisible(!basketVisible)
-  }
+    setBasketVisible(!basketVisible);
+  };
 
-
-  const addToBasket = (cat) =>{
+  const addToBasket = (cat) => {
     //function which is called by the search component to add the
     // cat to the basket.
     // The cat is passed as an object
 
     let tempBasket = [...basketItems];
-    tempBasket.push(cat)
-
-    setBasketItems(tempBasket)
-
-  }
+    tempBasket.push(cat);
 
+    setBasketItems(tempBasket);
+  };
 
   return (
     <>
-      <Basket visible={basketVisible} toggle={toggleBasket} basketState={basketItems}/>
-      <div>
-        <div className="nav1">
-          <p>All Products</p>
-          <input type="text"></input>
-          <ul>
-            <a href="#">Account</a>
-            <a onClick={toggleBasket}>Cart</a>
-          </ul>
-        </div>
-        <Search callback={addToBasket}/>
-        <Footer />
+      <div className="nav1">
+        <img className="cat" src={CatsIcon} alt="cat"></img>
+        <span>Cats4Lyf</span>
+        <input type="text"></input>
+        <img className="searchlogo" src={SearchIcon} alt="sl"></img>
+
+        <img
+          className="cartlogo"
+          onClick={toggleBasket}
+          src={CartIcon}
+          alt="cart"
+        ></img>
       </div>
+      <Basket
```

Comment:
Why did you move the Basket to the end of the html rather than the beginning?

Reply:
this is an old one, please check the latest one, thanks